co-simulation/bundle/src/assembly/resources/scenarios/Town05/sumo/Town05.sumocfg

```diff
@@ -1,14 +1,19 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <configuration xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://sumo.dlr.de/xsd/sumoConfiguration.xsd">
     <traci_server>
-      <remote-port value="2010"/> 
-      <num-clients value="1" />
+      <remote-port value="2010"/>
+      <num-clients value="2" />
```

Comment:
@Cheng-Yuan-0124  Don't we have these changes already reflected in CARMA Config? What is the appropriate place to add these changes? In CARMA Config they seemed to be associated with random traffic.

Reply:
In my latest carma-config PR, I updated the Town05.sumocfg file to reflect this parameter. However, I think it’s better to keep `num-clients` set to 1 here. By keeping it at 1, we ensure the configuration is optimized for running CDASim alone, allowing it to operate smoothly without requiring EVC-SUMO. In the carma-config PR, we set `num-clients` to 2 because carma-config launches EVC-SUMO, which needs the second client to function properly